### DISCUSSION TITLE
`df-print: paged` table has very large font size in revealjs

### DISCUSSION THREAD

**Discussion by ArthurAndrews** [quarto-dev/quarto-cli#3713] [Q&A]
I'm trying to print data frames in a revealjs presentation.  With default settings, the font sizes are large and the printed table only shows about 5 rows and 4 columns.  

Is this the intended behavior?  Any ideas how to fix?  

Many thanks!

```r
---
title: "test"
author: "arthur"
format: revealjs
df-print: paged
editor: source
---

## test table

```{r}
mtcars
```
```

![image](https://user-images.githubusercontent.com/65186717/208518969-89205b4e-0932-44a5-bfe9-90654e9ff7a7.png)

**Comment by ArthurAndrews** [comment]
![image](https://user-images.githubusercontent.com/65186717/208521358-70f43056-dcc6-48f8-b784-64f34716bfa4.png)

**Comment by ArthurAndrews** [comment]
I tried to add a style sheet, but it didn't have an effect on the presentation.  I'm out of my element :/


```

---
title: "css demo"
format: revealjs
editor: visual
css: my_style.css
df-print: paged
---

## Quarto

Text
```{r}
mtcars
```_
```

my_style.css
```
body {
  font-size: "12px";
}
```

**Reply by cscheid** [reply]
Are you on the latest quarto version? `quarto --version` will tell you. On the latest version of quarto, `df-print: paged` should be adapting the size to your screen. This is what I see on your file for small heights:

<img width="1724" alt="image" src="https://user-images.githubusercontent.com/285675/208687261-04342a76-7554-4e41-bfd0-50aa5ecb8c8c.png">

and for small widths:

<img width="498" alt="image" src="https://user-images.githubusercontent.com/285675/208687311-02da9228-9dc6-4c1f-9e8a-320a86e45fa4.png">

**Comment by ArthurAndrews** [comment]
Thanks!  I am in the habit of updating my packages, but not the quarto CLI.  I can see a similar result.  

Is there any way for it to scale based on number of rows.  If I give it more than 10 rows, it starts to run off the bottom of the slide.

```
## bigger table
```{r rows.print = 20}
mtcars
```X
```

![image](https://user-images.githubusercontent.com/65186717/208701418-c19c46dd-da2c-4f77-8f3d-4294e2a4881f.png)

**Reply by cscheid** [reply]
That'll be hard for us to make work in general. You'll need to learn to use custom CSS of yours to do it.
